libPhoneNumber/NBPhoneNumberUtil.m

```diff
@@ -2622,6 +2622,61 @@ - (BOOL)isPossibleNumber:(NBPhoneNumber*)number
 }
 
 
+- (NBEValidationResult)testNumberLength:(NSString *)number metadata:(NBPhoneMetaData *)metadata {
```

Comment:
can we name this something other than test*? By naming it with test it appears at first glance to be a unit test.